fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
+
+        final Graph graph =
+                container.getChildren().next().getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
```

Comment:
You _are_ assuming that. You are only testing one child, the first one to be returned by `next()`. That's the "first child". If that happens not to be the binary-child you created then the test will fail even though the software may be operating correctly. It would be stronger to retrieve all the children and filter to the one that has the name you gave your binary-child, then check _it_ for the title you think it should have. Let's leave it at that.